go.mod

```diff
@@ -1,17 +1,18 @@
 module github.com/openshift/runtime-utils
 
-go 1.12
+go 1.13
 
 require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/containers/image v3.0.2+incompatible
 	github.com/opencontainers/go-digest v1.0.0-rc1 // indirect
 	github.com/opencontainers/image-spec v1.0.1 // indirect
 	github.com/openshift/api v3.9.1-0.20191002160657-d92789481b05+incompatible
```

Comment:
this version needs fixing as per https://github.com/openshift/openshift-controller-manager/pull/85#discussion_r408247518

Reply:
version updated to matched to what we current have in https://github.com/openshift/openshift-controller-manager/pull/85